app/utils/email-tools.js

```diff
@@ -25,7 +25,7 @@ const validEmail = email => {
   const match = reValidEmail.exec(email)
   return {
     'local-part': match !== null ? match[1] : false,
```

Comment:
curious why `local-part` isn't updated to remove the `'` as per `domain` below, is it because of the `-`?

Reply:
yeh it will be because of that